src/rdds/lib/logging/get_logger.py

```diff
@@ -0,0 +1,13 @@
+import logging
+
+from . import LOGGING_FORMAT
+
+
+def get_logger(name: str = None) -> logging.Logger:
```

Comment:
```suggestion
def get_logger(name: str = None) -> Logger:
```
Import submodule directly to avoid long type hints and make the code cleaner and more readable